.changeset/config.json

```diff
@@ -10,6 +10,13 @@
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
   "ignore": ["@blockprotocol/site", "@crates/*", "@local/*"],
+  "fixed": [
+    [
+      "@blockprotocol/type-system",
+      "@blockprotocol/type-system-node",
+      "@blockprotocol/type-system-web"
+    ]
+  ],
```

Comment:
- https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#fixed-array-of-arrays-of-package-names
- https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md